lib/route.coffee

```diff
@@ -207,8 +207,15 @@ class @Route
     # Send response
     endpointContext.response.writeHead statusCode, headers
     endpointContext.response.write body
-    endpointContext.response.end()
-
+    switch statusCode
+        when 401, 403
+            # This is a simple, but not really adequate way to slow down unauthorized scans of the server.
+            # It is not sufficient because hackers can DDOS across multiple threads and systems.
```

Comment:
I think we discussed a middleware solution.
I haven't had time to look at that.
This code is sort of a stop gap measure...